src/test/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/service/testData.kt

```diff
@@ -42,7 +42,7 @@ fun prisonerAtLocation(bookingId: Long = 1234567, offenderNo: String = "A1234AA"
 fun prisonerIepLevel(
   bookingId: Long,
   iepCode: String = "STD",
-  reviewTime: LocalDateTime = LocalDateTime.now(),
+  reviewTime: LocalDateTime,
```

Comment:
clocks preset in tests were not passed in and ended up creating data with a different value for "now"